### ISSUE TITLE
Paren Issues for Arithmetic Rules.

### ISSUE THREAD

**Issue by eriq-augustine** [eriq-augustine/psl#56]
```
(0.5 * SameUser(A,B) + 0.5 *  SameUser(B,C)) >= SameUser(A,C)
```
Throws a lex error.

```
0.5 * SameUser(A,B) + 0.5 *  SameUser(B,C) >= SameUser(A,C)
```
Works fine.

**Comment by eriq-augustine** [comment]
This is doable, but non-trivial.
It will require changing arithmeticRuleExpression and a decent chunk of terse code:
https://github.com/eriq-augustine/psl/blob/develop/psl-parser/src/main/java/org/linqs/psl/parser/ModelLoader.java#L375

Which is a good thing, I never liked that code.

**Comment by eriq-augustine** [comment]
Fixed in 7a87d0c0c7f9fbd8109ff7e2750eab1907f7eea6.
